_posts/2018-05-24-open-source-hackillinois-2018.markdown

```diff
@@ -0,0 +1,45 @@
+
+​	Afterward, hacking began! Students began learning the ropes of Open Source and finding issues to solve. Many mentors had tagged issues for HackIllinois beforehand and were prepared to guide students through the initially intimidating process of understanding and improving a codebase.
+
+![Students start to work with mentor Andy Schwartzmeyer on the Apache Mesos project](/assets/article_images/open-source-hackillinois-2018/powershell.jpg "mesos")
+
+Last year, hacker Trent Spice worked with the Rust team and [won the Long Con prize](https://blog.hackillinois.org/spotlight/2017/09/29/open-Source-at-hackillinois-rust-cookbook.html) for HackIllinois 2017. This year, he came back with other students from Indiana University–Purdue University Indianapolis (IUPUI) and continued to work on Rust with mentors Steve Klabnik and Ashley Williams. HackIllinois aims to build these long-term relationships between students and the Open Source community, and our staff is super glad that groups from HackIllinois are still working together to contribute to and build on projects like [Rust Cookbook](https://github.com/rust-lang-nursery/rust-cookbook) and [RustBridge](https://github.com/rustbridge).
+
+![Hackers from IUPUI work with mentors from Rust in the ECEB](/assets/article_images/open-source-hackillinois-2018/rust.jpg "rust")
+
+We also had some awesome mentors representing Open Source projects from the fields of Data Science. We had one of the original founding members of [Julia](https://github.com/JuliaLang/julia), Jeff Bezanson, join us to show students the world of technical computing. Of course, we also had mentor Carol Willing (past Director of the Python Software Foundation) returning from HackIIllinois 2017 to work on [Jupyter](https://github.com/jupyter/jupyter).
+
+# Projects
+
+Tons of Open Source projects found themselves welcoming new contributors at HackIllinois. We had representatives from large Open Source projects such as [Apache Mesos](https://github.com/apache/mesos), [NPM](https://github.com/npm/npm), [Rust](https://github.com/rust-lang/rust), [Conda](https://github.com/conda/conda), [Homebrew](https://github.com/Homebrew/brew), and [Hyper](https://github.com/zeit/hyper). These influential projects gave students and mentors the opportunity to work together to contribute to some of the most common tools they use everyday.
+
+HackIllinois was home to lots of smaller, growing Open Source projects, too, like [OpenTreeMap](https://github.com/OpenTreeMap/otm-core), [NodeSchool](https://github.com/nodeschool/nodeschool.github.io) and [PrairieLearn](https://github.com/PrairieLearn/PrairieLearn).  These Open Source projects have immense opportunities for growth at HackIllinois. In fact, mentor Michael Maurizi told me that he and his team solved every issue that he had set aside for HackIllinois 2018!
```

Comment:
Link for Michael. Also I'd rephrase to "In fact, one of our mentors, Michael Maurizi, ..." or something similar.